lib/ember-cli/asset_resolver.rb

```diff
@@ -0,0 +1,53 @@
+module EmberCLI
+  class AssetResolver
+    def initialize(options)
```

Comment:
> What are your opinions on library support for older rubies?

Older rubies, your take! Depends much on our users indeed.

For Rails it is easier: Rails 3.1 and 4.0 are not supported, and have known vulnerabilities. I'd recommend dropping them, and users can keep using previous ember-cli-rails releases. Before the release of v1 I'd consider dropping 3.2 too, as it's close to real EOL as well.